src/components/Approval/Approval.tsx

```diff
@@ -0,0 +1,166 @@
+import { Button, Divider, Flex, Image, Link, SkeletonCircle } from '@chakra-ui/react'
+import get from 'lodash/get'
```

Comment:
I disagree with this, now if someone needs to import another lodash function we need an entire like for it.  It seems counter intuitive to nitpick code this much

Reply:
`get` isn't even needed. You should be using `?.` instead of `get`.